libs-scala/scala-utils/src/main/scala/com/daml/scalautil/nonempty/NonEmpty.scala

```diff
@@ -85,9 +85,27 @@ object NonEmptyColl extends NonEmptyCollInstances {
   }
 
   implicit final class ReshapeOps[F[_], A](private val nfa: NonEmpty[F[A]]) extends AnyVal {
+
+    /** See [[NonEmptyF]] for further explanation. */
     def toF: NonEmptyF[F, A] = NonEmpty.equiv[F, A](nfa)
   }
 
+  implicit final class UnReshapeOps[F[_], A](private val nfa: NonEmptyF[F, A]) extends AnyVal {
+
+    /** `x.toNE` is `(x: NonEmpty[F[A]])` but possibly shorter.  If code
+      * compiles without the call to `toNE`, you don't need the call.
+      */
+    def toNE: NonEmpty[F[A]] = nfa
```

Comment:
How about calling this `fromF`? It is the inverse operation of `toF` and `toF` doesn't mention `NE` either.

Reply:
`fromNEF` 39fcfc912e5

`toNEF` 7d675c5cccb